apps/academy/src/pages/fundamentals.tsx

```diff
@@ -0,0 +1,62 @@
+import type { FC } from "react";
+import React from "react";
+import { type NavItem, TopBar, TrackCard } from "ui";
+
+import { ConnectButton } from "@/components/ConnectButton";
+
+const sampleMenus: NavItem[] = [
+  {
+    name: "Fundamentals",
+    href: "/fundamentals",
+    icon: "clarity_blocks",
+  },
+  {
+    name: "Tracks",
+    href: "/",
+    icon: "vector",
+  },
+  {
+    name: "Community",
+    href: "/",
+    icon: "dd_logo",
+  },
+];
+
+const Fundamentals: FC = () => {
+  const trackCards = Array.from({ length: 10 }).map(() => ({
+    imgSrc: "/eth_fam.png",
+  }));
+
+  return (
+    <div className="flex h-screen w-full flex-col bg-black lg:flex-row">
+      <div className="flex h-full min-h-screen flex-1 flex-col items-center justify-between bg-[url('/fundamental-bg.jpeg')] bg-cover bg-no-repeat object-center lg:fixed lg:inset-y-0 lg:w-1/2">
+        <div className="mt-10">
+          <TopBar menus={sampleMenus} />
```

Comment:
use the layout component for this
the Topbar menu is included https://github.com/Developer-DAO/academy-turbo/blob/b0163eed1151a91ee4a14ed1433823a2361f36f2/apps/academy/src/components/Header.tsx#L28

Reply:
@okhaimie-dev I think you can still make it work while using the layout. 

I updated the way the layout header is displayed on my last PR #33

Some steps I think could work 
- Position the fundamentals page as grid
- make the height and width take 100% of the viewport
- make the second grid col scrollable and wrap your contents in it


Some trade off being the footer will be visible.

@Markkos89 If possible I think the footer should be decoupled from the layout as only the home page uses it to allow the approach above work.

Reply:
> @okhaimie-dev I think you can still make it work while using the layout.
> 
> I updated the way the layout header is displayed on my last PR #33
> 
> Some steps I think could work
> 
> * Position the fundamentals page as grid
> * make the height and width take 100% of the viewport
> * make the second grid col scrollable and wrap your contents in it
> 
> Some trade off being the footer will be visible.
> 
> @Markkos89 If possible I think the footer should be decoupled from the layout as only the home page uses it to allow the approach above work.

Loving this, thanks a lot @peteruche21 !

@okhaimie-dev can follow peter's comments? We will appreciate it.

@peteruche21 regarding the footer not beeing in all pages on the designs, can we include it anyways? It's my opinion, for me makes sense to it be present on all pages, what do you guys think? cc @okhaimie-dev @wolovim @elPiablo

Reply:
Having the Footer show up in the Fundamental page, will throw off the design done in Figma for Fundamentals and Tracks page.

![CleanShot 2023-10-27 at 17 58 27](https://github.com/Developer-DAO/academy-turbo/assets/57156589/4351c9a6-08a1-4f74-a2bb-f3ad1b4144fb)

So, just to clarify @Markkos89 

1. You are asking me to go off the design done by @uma-l, and proceed to include the Footer to the fundamental page?

This will span across both cols (left & right)? And will I be able to make it stick to the bottom of the left and right cols while still keeping the right col scrollable. = remember the left col is fix = I reckon it will only leave a small view space for the fundamental lessons to the right

![CleanShot 2023-10-27 at 18 14 36](https://github.com/Developer-DAO/academy-turbo/assets/57156589/3316b443-7bee-4fe4-93af-1eb6e83dbedf)

Reply:
@okhaimie-dev was my idea but, for the sake of moving forward, please submit the PR without the footer. We can then update the page IF the footer will be included.

Thank you Okhai!!! :rocket: 

Best regards